packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -326,27 +326,31 @@ class FormAds extends Component {
             default_advert_description: [v => !v || lengthValidator(v), v => !v || textValidator(v)],
             max_transaction: [
                 v => !!v,
```

Comment:
```suggestion
                v => !!v || !isNaN(v),
```

Reply:
It can't be this way. We're using different error message for each case.